src/frontends/onnx/tests/convert_partially_tests.cpp

```diff
@@ -0,0 +1,110 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <file_utils.h>
+#include <gtest/gtest.h>
+
+#include <string>
+
+#include "common_test_utils/file_utils.hpp"
+#include "common_test_utils/ngraph_test_utils.hpp"
```

Comment:
Is it required?

Reply:
yes, `file_utils.hpp` is needed for `getModelFromTestModelZoo`, `ngraph_test_utils.hpp` for `count_ops_of_type`, but `#include "common_test_utils/ngraph_test_utils.hpp"` can be removed from `convert_tests.cpp`